### PR TITLE
test: Ensure all tests have an appropriate sig prefix

### PR DESCRIPTION
**Pull request by smarterclayton**
SIG names were chosen to match the upstream (apps is workloads).
Where no upstream exists new ones were defined (etcd|devex).

### CODE REVIEW

File: cmd/openshift-tests/e2e.go [hunk 1]
```diff
@@ -93,7 +93,7 @@ var staticSuites = []*ginkgo.TestSuite{
 		Tests that exercise language and tooling images shipped as part of OpenShift.
 		`),
 		Matches: func(name string) bool {
-			return strings.Contains(name, "[image_ecosystem]") && !strings.Contains(name, "[Local]")
+			return strings.Contains(name, "[Feature:ImageEcosystem]") && !strings.Contains(name, "[Local]")
```
Comment (bparees) [inline cmd/openshift-tests/e2e.go]: sig-samples ?
sig-imagesamples ?
sig-imageecosystem ?

**Comment by bparees** [comment]
do we have a canonical list of sigs -> team mappings somewhere?

also i would have voted for things like "sig-builds" instead of "sig-devex".  makes it easier to move components between teams if we were so inclined.

**Comment by smarterclayton** [comment]
I will accept a follow on PR

**Pull request merged**